tests/test_utils.py

```diff
@@ -1038,7 +1045,7 @@ def test_index_has_overlap(obj, expected):
         (
             [
                 pd.Index([1, np.nan], dtype='Int64', name='idx'),
-                pd.Index([1, 2, 3], name='idx'),
+                pd.Index([1, 2, 3], dtype='Int64', name='idx'),
```

Comment:
I this needed now? Cause the test was supposed to show that you can `intersect()` a `int64` with `Int64`.

Reply:
So maybe change the dtype of the result to `int64`.

Reply:
You can, but it will then return the strange `Int64Index` and not `Index`. I can also change the return value or I can change the order to
```python
            [
                pd.Index([1, 2, 3], name='idx'),
                pd.Index([1, np.nan], dtype='Int64', name='idx'),
            ]
```
which would also fix the return value.

Reply:
Ah actually it's a bit more complicated that that:

```
objs = [
    pd.Index([1, np.nan], dtype='Int64', name='idx'),
    pd.Index([1, 2, 3], name='idx'),
]
audformat.utils.intersect(objs)
```
```
Int64Index([1], dtype='int64', name='idx')
```

but:

```python
objs = [
    pd.Index([1, 2, 3], name='idx'),
    pd.Index([1, np.nan], dtype='Int64', name='idx'),
]
audformat.utils.intersect(objs)
```
```
Index([1], dtype='Int64', name='idx')
```

So the `dtype` of the result depends on the order of the objects.

Reply:
Exactly

Reply:
I updated the test and now have each orders covered.

Reply:
So maybe we should test for both orders and use `pd.Index([1, 2, 3], name='idx')` instead of `pd.Index([1, 2, 3], dtype='Int64', name='idx')`.

Reply:
We test now for both orders.